applications/Tenure_and_Resource_Stewardship_Branch.md

```diff
@@ -0,0 +1,151 @@
+
+## 1.5. Administering Authority
+
+[Energy, Mines and Digital Trust (EMDT)](https://digital.gov.bc.ca/case-studies/emdt) is the Administering Authority on behalf of Tenure during the pilot phase of development.
+
+* 	**Name:** Kyle Robinson 
+* 	**Title:** Senior Strategic Advisor, Digital Trust Ecosystems
+* 	**Organization:** Brairtech Consulting Inc.
+* 	**Email:** kyle.robinson@brairtech.ca
+
+## 1.6. Purpose
+
+The purpose of this governance framework is to describe the rules/policies/procedures for verifiable credential exchanges involving the Tenure and Resource Stewardship Branch (TRSB) with the open global community. The purpose of the rules is to enable all actors to understand agreed upon standards, terminology and processes that allow the community to interact with TRSB in a trusted manner. This will help determine a governing framework and operating model for a global ecosystem that identifies how credentials can be issued, held, and verified. 
+
+## 1.7. Scope 
+The Tenure and Resource Stewardship Branch (TRSB) is a participant in an open ecosystem and the focus of this framework is to describe the process Tenure uses for digital exchanges on Indy networks.
+
+## 1.8. Objectives
+
+TBD
+
+## 1.9. Principles 
+
+TBD
+
+## 1.10. General Requirements
+TBD
+
+## 1.11. Revisions
+TBD
+
+## 1.12. Extensions
+
+There are no extensions to this Governance Framework.
+
+## 1.13. Schedule of Controlled Documents
+TBD
+
+# 2. Controlled Documents
+
+## 2.1. Glossary
+TBD
+
+## 2.2. Risk Assessment
+
+TBD
+
+## 2.3. Trust Assurance and Certification
+
+TBD
+
+## 2.4. Governance Requirements
+
+## 2.5. Business Requirements
+
+## 2.5.1. Establishment of Connection
+
+1. The Tenure and Resource Stewardship Branch (TRSB) MUST send an invitation to the other business entity via email to initiate the exchange of information. 
+2. The receiving party MUST use the invitation in order to establish a secure connection.
+
+## 2.5.2. Issuance of BC Petroleum & Natural Gas Title Credential 
+
+1. Using the established connection, TRSB MUST receive a credential request from the client.
+2. Continuing from step 1, the client MUST provide attributes in the credential request as outlined in the [BC Petroleum & Natural Gas Title Credential Governance](https://github.com/bcgov/bc-vcpedia/blob/main/credentials/credential-bc-petroleum-natural-gas-title.md)
+4. TRSB MUST receive and review the request
+5. Any clarification questions MAY be asked via the secure messaging function 
+6. After completion of verification activity, Tenure MUST update any attribute values as necessary
+7. TRSB SHALL offer the credential to the client 
+8. Client MAY choose to accept or negotiate the credential to request changes
+
+
+## 2.6. Technical Requirements
+
+## 2.6.1. Public DID of Tenure
```

Comment:
The Public DID should be listed here - not related to Aries

Reply:
![image](https://user-images.githubusercontent.com/122049466/225963000-3a4bf94c-2c64-49bf-bcba-0dca983c4357.png)